src/types/shop.ts

```diff
@@ -1,16 +1,13 @@
 export type Product = {
   name: string
-  price: Nimiq.Transaction
-}
-
-export type OrderMeta = {
-  sender: Nimiq.Address
+  price: number //TODO(svub) Q: was `Nimiq.Transaction` before. Don't understand. (?)
 }
 
 export type Order = {
-  price: number
+  // TODO(svub) Q: price is part of product >> would consider it redudant info here (?)
+  //price: number
   products: Product[]
-  meta: OrderMeta
+  meta: string
```

Comment:
`string`? Why does not `OrderMeta` make sense anymore?

Reply:
Meta data is defined by the shop owner. It can be anything they need to fulfill the order. The shop owner has to take care of collecting the data from the customer and then serialize the information. That's at least my POV. :)

Thus, I was wondering, a string? Or even just an object as the entire thing will be run through `JSON.stringify()` anyhow. I tend to make it an object - or actually `any` in TS.

Reply:
See https://github.com/svub/nimiq-popup-shop/pull/13#discussion_r339356183